atexit.go

```diff
@@ -30,14 +30,16 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"sync"
 )
 
 const (
 	// Version is package version
-	Version = "0.1.0"
+	Version = "0.2.0"
 )
 
 var handlers = []func(){}
+var handlersLock sync.RWMutex
```

Comment:
I don't see the value of `RWLock` here. If someone adds a handler while we exit - it won't get executed anyway.

Reply:
RWLock ensures that none of other goroutines adds another handler at the same time and also ensures that handlers do not run while another handler is being added.

Reply:
I know. Seems like a overkill since the handlers are going to be read once.